libs/styles/styles.css

```diff
@@ -4,7 +4,7 @@
   --global-height-nav: 64px;
   --global-height-breadcrumbs: 33px;
   /* stylelint-disable-next-line custom-property-pattern */
-  --global-height-navPromo: 65px;
+  --global-height-navPromo: 72px;
```

Comment:
Have there been any considerations regarding the two-line, 80px use-case utilizing a min-height here?

Reply:
I noticed it spawns on two lines between 1200 and 1270px

Reply:
@narcis-radu is right, the Promo spans two rows between 1200 and 1287px. Either the copy needs to be adapted or some styles.

Reply:
Yes. Based on discussions with _Joel Blytt_ the character count limit in design specs will be updated so that promobar displays on single line for this range also.